web/cypress/integration/plugin/create-edit-delete-plugin.spec.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('e2e test for plugin page', () => {
+  const sleepTime = 100;
+
+  beforeEach(() => {
+    // init login
+    cy.login();
+  });
+
+  it('should enable two plugins', () => {
+    //  go to route plugin list page
+    cy.visit('/');
+    cy.contains('Plugin').click();
```

Comment:
can we remove `cy.wait()` and write like this?

```suggestion
    cy.contains('Plugin').click({timeout: xxx});
```

Reply:
fixed, thanks for reminding.